application/go_inc/helpers/helper__functions.php

```diff
@@ -78,7 +78,17 @@ function go_get_menu_items($menu_id) {
         foreach($query->result() as $result){
             if($count == 1) {
                 $return .= "<div id='menu-id-" . $result->MenuID . "'>"; // setup parent DIV and UL
-                $return .= "<ul>"; 
+
+                //$return .= "<ul>"; 
+                
+                if($menu_id == 1){
```

Comment:
We can't manage it by only menu ids for multiple menus . Actually we have two types of menu first top menu and second one is left menu. I can create dynamic css and html class with ids like that :
$return .= "<ul class='top-menu-ul-" . $menu_id . ">";
$return .= "<ul class='side-menu-ul-" . $menu_id . ">";
Is that ok ?